packages/tocco-ui/src/StatedValue/StatedValue.stories.js

```diff
@@ -98,11 +98,8 @@ storiesOf('Tocco-UI | StatedValue', module)
         }),
         label: text('label', 'label'),
         mandatory: boolean('mandatory', false) || undefined,
-        immutable: boolean('immutable', false) || undefined,
-        type: select('type', {
-          'display': 'display',
-          'field': 'field'
-        }, 'display')
+        isDisplay: boolean('isDisplay', false) || undefined,
```

Comment:
this `|| undefined` was originally written to copy the code nicely in the story. Means
`<Test isDisplay="false"/> ` will be changed to `<Test />`
and `<Test isDisplay="true"/> `  stays the same.
Since a wrapper component is used its not possible to copy the code. So the `|| undefined`  can be omitted.